### PR TITLE
Parameters.datetime

### PR DESCRIPTION
**Pull request by esjee**
Hey, I implemented this for our fork and figured you might find it useful. What do you think of this change?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,7 +164,8 @@ ActionController::Parameters.action_on_invalid_parameters = :log
 |--------------------------------|-------------------------------------------------------------------------|
 | Parameters.string              | value.is_a?(String)                                                     |
 | Parameters.integer             | value.is_a?(Fixnum) or '-1'                                             |
-| Parameters.float               | value.is_a?(Float)  or '-1.2'                                           |
+| Parameters.float               | value.is_a?(Float) or '-1.2'                                            |
+| Parameters.datetime            | value.is_a?(DateTime) or '2014-05-13'                                   |
```
Comment (staugaard) [inline README.md]: `or '2015-03-31T14:34:56Z'`

File: test/date_constraint_test.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require_relative 'test_helper'
+
+describe 'date parameter constraints' do
+  subject { ActionController::Parameters.datetime }
+
+  permits "2015-03-31", as: DateTime.parse("2015-03-31")
+
+  rejects "2015-03-32"  # Invalid day
```
Comment (staugaard) [inline test/date_constraint_test.rb]: Also test time parsing `2015-03-31T14:34:56Z`

File: README.md (hunk 1 above)
Comment (esjee) [inline README.md]: Not sure if you want this as a replacement, or as an addition to the date version. I think the latter makes more sense, so went with that.

**Comment by esjee** [comment]
Hey @staugaard , thanks for your comments. I think I've addressed them, could you have another look?

**Comment by staugaard** [comment]
:+1:

**Pull request merged**